editor/src/components/canvas/context-menu-state.tsx

```diff
@@ -0,0 +1,47 @@
+import { atom, useAtom } from 'jotai'
+
+import { BodyMenuOpenClass } from '../../core/shared/utils'
+import React from 'react'
+
+export const ContextMenuStateAtom = atom([] as string[])
```

Comment:
in general if we can use the generic parameter `<P>`, we prefer it to casting with `as P` for increased type safety.

Even minorer, but we use `Array<string>` instead of the `string[]` notation.

```suggestion
export const ContextMenuStateAtom = atom<Array<string>>([])
```